src/main/resources/application.properties

```diff
@@ -1,4 +1,12 @@
 security.sessions=NEVER
 
 multipart.maxFileSize=15MB
-multipart.maxRequestSize=15MB
\ No newline at end of file
+multipart.maxRequestSize=15MB
+
+spring.datasource.url=jdbc:mysql://localhost/trainings
+spring.datasource.username=exadel
+spring.datasource.password=exadel
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+
```

Comment:
datasource was configured in java class according to security. No need to duplicate it there